### DISCUSSION TITLE
iOS app can't download or share PDF file

### DISCUSSION THREAD

**Discussion by oliversturm** [community/community#5695] [Mobile]
I'm trying to use the iOS GitHub app on my iPad. In my repo there are various PDF files that I want to share with local Acrobat reader so they remain available offline. I have done this for years using the app "Branches" which has mysteriously disappeared recently.

I open the PDF directly within the GitHub app. I then use the ellipsis button in the top right corner and select "Share via...". Numerous apps and "actions" are included in the popup menu, but Adobe Acrobat is not one of them for some reason. I know there are "edit actions" and other "..." buttons to configure the entries in the lists, but Acrobat is still not there. In addition, other apps I tried don't work correctly - for instance I have "Copy to PDF Expert", but when I select this, PDF Expert simply opens without the file and nothing is shared to it. I tried "Open in Safari", "Copy to FileBrowser Pro" and a few others, but none of them did anything useful with the file.

In summary:

* What share options there are don't seem to work correctly for a PDF file, as far as my tests went
* Acrobat is missing for unknown reasons
* There is no "download" or other obvious action to simply say "get the file to my iPad for further local handling".

I'm sure I must be missing something, but I wonder what it is!

Update: I tried the app Working Copy just now, and the action "Import to Acrobat" appears there just like I would expect.
